src/mpyl/steps/deploy/k8s/chart.py

```diff
@@ -460,13 +460,29 @@ def to_sealed_secrets(self) -> V1SealedSecret:
 
     @staticmethod
     def _to_resources(resources: Resources, defaults: ResourceDefaults, target: Target):
```

Comment:
should we maybe rename this function? it's a bit weird to have a `to_resources` function has an input param `resources`